interface/forms/newpatient/common.php

```diff
@@ -227,7 +227,7 @@ function cancelClickedOld() {
                         <legend><?php echo xlt('Visit Details')?></legend>
                         <div id = "visit-details">
                             <div class="form-group ">
-                                    <label for="pc_catid" class="control-label col-sm-2 text-right"><?php echo xlt('Visit Category:'); ?></label>
+                                    <label for="pc_catid" class="control-label col-sm-2 text-to-side"><?php echo xlt('Visit Category:'); ?></label>
```

Comment:
hi @matrix-amiel ,
Wouldn't it make more sense to call this text-to-right?
(of which, it is assumed that rtl will do the opposite)
(then allowing another class, text-to-left that does the opposite)

Reply:
note i changed it to `oe-text-to-right`

Reply:
Agree.. done.